### PR TITLE
Metaphysics v2 migration

### PR DESCRIPTION
**Pull request by alloy**
Closes https://artsyproduct.atlassian.net/browse/PLATFORM-1446

This is the PR for the `mp-v2-migration` branch that all other PRs should target.

### CODE REVIEW

**Comment by zephraph** [comment]
It’s beautiful! ❤️

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: package.json [hunk 1]
```diff
@@ -33,26 +36,21 @@
     "prettier-project": "yarn run prettier-write 'src/**/*.{ts,tsx}'",
     "prettier-check": "prettier --check 'src/**/*.{ts,tsx}'",
     "prettier-write": "yarn run prettier --write",
-    "relay": "node verify-node-version.js && relay-compiler --src ./src --schema data/schema.graphql --language typescript --artifactDirectory ./src/__generated__ --exclude '**/node_modules/**,**/__mocks__/**,**/__generated__/**'",
+    "relay": "relay-compiler",
```
Comment (damassi) [inline package.json]: nice 👍

File: package.json [hunk 2]
```diff
@@ -33,26 +36,21 @@
...
     "release": "auto shipit",
     "start": "yarn storybook",
-    "storybook": "node verify-node-version.js && concurrently --raw --kill-others 'yarn relay --watch' 'start-storybook --quiet -s ./public -p 9001'",
-    "sync-schema": "curl https://raw.githubusercontent.com/artsy/metaphysics/master/_schema.graphql -o data/schema.graphql; yarn prettier --write --parser graphql data/schema.graphql",
+    "storybook": "concurrently --raw --kill-others 'yarn relay --watch' 'start-storybook --quiet -s ./public -p 9001'",
+    "sync-schema": "curl curl https://raw.githubusercontent.com/artsy/metaphysics/master/_schemaV2.graphql -o data/schema.graphql; yarn prettier --write --parser graphql data/schema.graphql",
```
Comment (damassi) [inline package.json]: `curl curl`

File: src/Apps/Auction/Components/LotInfo.tsx [hunk 3]
```diff
@@ -47,20 +47,20 @@ export const LotInfo: React.FC<Props> = ({ artwork, saleArtwork }) => {
 export const LotInfoFragmentContainer = createFragmentContainer(LotInfo, {
   artwork: graphql`
     fragment LotInfo_artwork on Artwork {
-      _id
+      internalID
       date
       title
       imageUrl
-      artistNames: artist_names
+      artistNames: artistNames
```
Comment (damassi) [inline src/Apps/Auction/Components/LotInfo.tsx]: Would it be complicated to evaluate whether each selection is identical before writing the addition?

Comment (alloy) [reply src/Apps/Auction/Components/LotInfo.tsx]: Hmm, I’m surprised by this, in most places we did check if the new name was different than what was there already. Anyways, it should be easy to write a little codemod to clean this up post-migration 👍

**Comment by mzikherman** [comment]
Ok, let's get this released into Reaction and out on staging!

**Comment by artsyit** [comment]
:rocket: PR was released in v22.0.0 :rocket:

**Pull request merged**